demos/python_demos/requirements.txt

```diff
@@ -1,3 +1,7 @@
-opencv-python
-numpy
-scipy
+joblib==0.14.0
+nibabel==2.5.1
+numpy==1.17.2 ; python_version >= "3.4"
+scikit-learn==0.21.3
+scipy==1.3.1
+six==1.12.0               # via nibabel
+tqdm==4.39.0
```

Comment:
This file should not pin versions. The demos are supposed to work with any recent versions of the dependencies.